gradio/components.py

```diff
@@ -2723,25 +2722,13 @@ def __init__(
         Parameters:
         value (Any): the initial value of the state.
         """
-        self.value = orjson.loads(value) if isinstance(value, str) else value
-        try:
-            orjson.dumps(self.value, option=orjson.OPT_SERIALIZE_NUMPY)
-        except TypeError:
-            raise ValueError(
-                f"Cannot set initial value because type: {type(value)} is "
-                "not JSON serializable. Instead, you can set the initial "
-                "value to be None, and in your Python function, change it "
-                "to be the your desired value when the function is "
-                "run initially."
-            )
+        self.value = deepcopy(value)
         self.stateful = True
         IOComponent.__init__(self, **kwargs)
 
     def get_config(self):
         return {
-            "value": orjson.dumps(self.value, option=orjson.OPT_SERIALIZE_NUMPY).decode(
-                "utf-8"
-            ),
+            "value": self.value,
```

Comment:
We may need to revert this change as well? This breaks this fix made last week https://github.com/gradio-app/gradio/issues/1408